src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -52,11 +55,16 @@ const defaultProps = {
 
 const OptionRowLHN = (props) => {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
+
+    React.useEffect(() => {
```

Comment:
I'm actually not seeing why this is necessary. I removed this and everything seems to work fine.